corehq/apps/hqwebapp/two_factor_gateways.py

```diff
@@ -134,19 +137,49 @@ def get_ip_address():
     return status != _status_accepted
 
 
-two_factor_setup_rate_limiter = RateLimiter(
```

Comment:
can you do a full text search for this word? I have a feeling there might be other places this is referenced (in code or in a comment)

Reply:
found an instance of it in the comment so I updated that. All other occurrences were part of the "global_two_factor_setup_rate_limiter"